src/pages/Settings/Settings.js

```diff
@@ -28,147 +29,113 @@ function useUpdateVersion () {
-      handleClose={() => setToggleModal(null)}
-      toggleModal={toggleModal}
+      handleClose={() => setModalVisible(null)}
+      modalVisible={modalVisible}
       availableVersion={availableVersion}
       updateVersion={updateVersion} />
-
-    <hr />
-    <hr />
-
-    <Button name='factory-reset' variant='danger' wide styleName='factory-reset-button' onClick={() => push('/factory-reset')}>Factory Reset</Button>
   </PrimaryLayout>
 }
 
-export function SettingsTable ({ updateAvailable, header, children }) {
-  return <table styleName='settings-table' data-testid='settings-table'>
-    <thead>
-      <tr key='heading'>
-        <th id={header.toLowerCase().trim()} styleName='settings-row-header'>
-          <h5 styleName='row-header-title'>{header}
-            { updateAvailable
-              ? <span styleName='second-header'> Update Available</span>
-              : null
-            }
-          </h5>
-        </th>
-      </tr>
-    </thead>
-    <tbody>
-      {children}
-    </tbody>
-  </table>
-}
-
-export function SettingsRow ({ label, content, dataTestId }) {
-  return <tr styleName='settings-row' data-testid='settings-row'>
-    <td styleName='settings-col align-left'>
-      <h3 styleName='row-label' data-testid='settings-label'>{label}</h3>
-    </td>
-    <td styleName='settings-col align-right'>
-      <h3 styleName='row-content' data-testid={dataTestId}>{content}</h3>
-    </td>
-  </tr>
-}
-
-export function SoftwareUpdateRow ({ label, showModal, updateAvailable }) {
-  return <tr styleName='settings-row' data-testid='settings-row'>
-    <td styleName='settings-col align-left'>
-      <h3 styleName='row-label' data-testid='settings-label'>{label}</h3>
-    </td>
-    <td styleName='settings-col align-right'>
-      { updateAvailable
-        ? <SoftwareUpdateButton showModal={showModal} />
-        : <h3 styleName='row-content'>Your Software is up-to-date</h3>
-      }
-    </td>
-  </tr>
+export function SettingsRow ({ label, value, dataTestId, bottomStyle }) {
+  return <div styleName={bottomStyle ? 'settings-row-bottom' : 'settings-row'} data-testid={dataTestId}>
+    {typeof label === 'string'
+      ? <span styleName='settings-label'>{label}</span>
+      : label}
+    {typeof value === 'string'
+      ? <span styleName='settings-value'>{value}</span>
+      : value}
+  </div>
 }
 
-function SoftwareUpdateButton ({ showModal }) {
+function VersionUpdateButton ({ showModal }) {
   return <Button
+    variant='white'
     onClick={showModal}
-    styleName='update-version-button'>
+    styleName='version-update-button'>
     Update Software
   </Button>
 }
 
-export function UpdateSoftwareModal ({ settings, handleClose, toggleModal, availableVersion, updateVersion }) {
+export function VersionUpdateModal ({ settings, handleClose, modalVisible, availableVersion, updateVersion }) {
   if (!availableVersion) return null
   const onYes = () => {
     updateVersion()
     handleClose()
   }
   return <Modal
```

Comment:
One style note.  The Yes and No buttons in the modal, still have the original link styling  (ie: blue with underline).  Please just update this one style. :+1: